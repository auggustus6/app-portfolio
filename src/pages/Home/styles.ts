/* eslint-disable prettier/prettier */
import styled from 'styled-components/native';

export const Container = styled.SafeAreaView`
  ${({theme}) => `
      flex:1;
      background:${theme.colors.white};
    `}
`;




