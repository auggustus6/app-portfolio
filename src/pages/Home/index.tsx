import React from 'react';
import {Text} from 'react-native';

import * as Styles from './styles';

const Home = () => {
  return (
    <Styles.Container>
      <Text>Ola Mundo</Text>
    </Styles.Container>
  );
};

export default Home;
