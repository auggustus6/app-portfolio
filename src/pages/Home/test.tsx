/* eslint-disable no-undef */

import React from 'react';
import Home from '.';

import {render} from '../../utils/tests/helpers';

it('should render screen correctly', () => {
  const {toJSON} = render(<Home />);

  expect(toJSON()).toMatchSnapshot();
});
