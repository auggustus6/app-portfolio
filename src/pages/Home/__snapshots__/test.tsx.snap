// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render screen correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {},
    ]
  }
>
  <Text>
    Ola Mundo
  </Text>
</RCTSafeAreaView>
`;
